workers/WorkerGlobalScope_requestAnimationFrame.tentative.worker.js

```diff
@@ -0,0 +1,19 @@
+importScripts("/resources/testharness.js");
```

Comment:
Is this needed? Seems weird that it's required for worker.js and not window.js.

Reply:
It is. One could file an issue about that; or perhaps we should remove `.worker.js`.

Reply:
Commented on #11529.